plugins/labhub.py

```diff
@@ -82,45 +82,49 @@ def invite_cmd(self, msg, match):
         invitee = match.group(1)
         inviter = msg.frm.nick
 
-        if invitee == 'me':
-            user = msg.frm.nick
-            response = tenv().get_template(
-                'labhub/promotions/newcomers.jinja2.md'
-            ).render(
-                username=user,
-            )
-            self.send(msg.frm, response)
-            self.TEAMS[self.GH_ORG_NAME + ' newcomers'].invite(user)
-            self.invited_users.add(user)
-            return
-
         team = 'newcomers' if match.group(2) is None else match.group(2)
+        team = team.lower()
+
+        is_developer = self.TEAMS[self.GH_ORG_NAME +
+                                  ' developers'].is_member(inviter)
+        is_maintainer = self.TEAMS[self.GH_ORG_NAME +
+                                   ' maintainers'].is_member(inviter)
 
         self.log.info('{} invited {} to {}'.format(inviter, invitee, team))
 
-        if self.TEAMS[self.GH_ORG_NAME + ' maintainers'].is_member(inviter):
-            valid_teams = ['newcomers', 'developers', 'maintainers']
-            if team.lower() not in valid_teams:
-                return 'Please select from one of the ' + ', '.join(valid_teams)
+        valid_teams = ['newcomers', 'developers', 'maintainers']
+        if team not in valid_teams:
+            return 'Please select from one of the valid teams: ' + ', '.join(
+                    valid_teams)
+
+        def invite(invitee, team):
             team_mapping = {
                 'newcomers': self.GH_ORG_NAME + ' newcomers',
                 'developers': self.GH_ORG_NAME + ' developers',
                 'maintainers': self.GH_ORG_NAME + ' maintainers'
             }
 
-            # send the invite
-            self.TEAMS[team_mapping[team.lower()]].invite(invitee)
+            self.TEAMS[team_mapping[team]].invite(invitee)
+
+        if is_developer and team == 'newcomers':
```

Comment:
Modify this to nested if statement, so that we need to give a particular error message when dev tries to invite members into dev or maintainers team.

Reply:
Also add the test case for the above.

Reply:
@Vamshi99 a maintainer is also a developer and when he invites someone to developers, it'll show the error message according to this^

Reply:
Nah, every maintainer isn't a developer, some people will leave dev team after getting promoted or they are directly made maintainer.
Example: coala-bot, fneu and many more

Reply:
Also, first check isMaintainer. To know why, check what happens when maintainer(dev team member) invites members to dev or maintainers team.

Reply:
done